jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   snapshotSerializers: ['enzyme-to-json/serializer'],
   transform: {
     '^.+\\.jsx?$': 'babel-jest',
-    '^.+\\.png$': '<rootDir>/test/transforms/file.js',
+    '(?!^.+\\.jsx?$)': '<rootDir>/test/transforms/file.js',
```

Comment:
🤷

Reply:
The `?!` is a so-called "negative lookahead". This'll match only if the negated expression does not match. If you look at what's inverted, it's exactly the `RegExp` on the preceding line. So this should match whenever the line above doesn't.